src/analysis/openstreetmap/qgsosmdownload.h

```diff
@@ -49,6 +49,7 @@ class ANALYSIS_EXPORT QgsOSMDownload : public QObject
     static QString queryFromRect( const QgsRectangle &rect );
 
     QgsOSMDownload();
+    QgsOSMDownload( const QString &query );
```

Comment:
Thanks @3nids. I was having a look too why travis was failing.

So for getters/setters, it's not compulsory to have docstring. But it's compulsory for constructors, right? I can see it's quite empty there https://qgis.org/api/classQgsOSMDownload.html

I noticed two kind of docstring: 
`//! My Docstring for a getter/setter`
and
```
 
    /** Constructor for QgsOSMDownload
     * \param query The query to execute in the Overpass API.
     *
     * \since QGIS 3.0
     */
```

The last one is preferred?
It's not on the developer guide :  https://www.qgis.org/en/site/getinvolved/development/qgisdevelopersguide/codingstandards.html